### ISSUE TITLE
VERY annoying horizontal scroll behaviour

### ISSUE THREAD

**Issue by Erik262** [iina/iina#3264]
Hello guys,

I'm using the latest IINA 1.1.2 build 128 on my mac with the magic mouse 2 and I deactivated horizontal scrolling in the settings BUT it still works!

If I want to skip any file, then I have to click twice or three times, because of this sensitive horizontal scrolling! 
So please fix this in the next update. It is maybe a nice feature, but definitely not for magic mouse user.

**Comment by edrozenberg** [comment]
Previously reported here https://github.com/iina/iina/issues/2996 and mentioned in a number of other closed issues.

Sure would be nice if this could be changed such that horizontal scroll could be absolutely disabled in all scenarios regardless of where the mouse cursor is - disturbing the user's position in the video via accidental touch or mousing is a bad thing.

**Comment by anohren** [comment]
> Previously reported here #2996 and mentioned in a number of other closed issues.
> 
> Sure would be nice if this could be changed such that horizontal scroll could be absolutely disabled in all scenarios

Correct me if I'm wrong but didn't you yourself close that very issue after concluding that nothing could be done? Is it not possible to just open it again?

Anyway, since there was no further response in that issue to my assertion to the contrary I have since distilled some problems with hover scrolling in a separate issue that is still open: https://github.com/iina/iina/issues/3102

**Comment by axu2** [comment]
EDIT: I am wrong

~Just reporting that this setting works for me on latest macOS Sonoma. Before changing this setting, touching the magic mouse would seek the video, now it does nothing, which is good. So I guess this can be closed.~

![image](https://github.com/user-attachments/assets/926aeb02-9b3d-42bd-b684-9f6cdf491ebe)

**Comment by Erik262** [comment]
> Just reporting that this setting works for me on latest macOS Sonoma. Before changing this setting, touching the magic mouse would seek the video, now it does nothing, which is good. So I guess this can be closed.
> 
> ![image](https://private-user-images.githubusercontent.com/20757319/353932798-926aeb02-9b3d-42bd-b684-9f6cdf491ebe.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI0NTIzMzMsIm5iZiI6MTcyMjQ1MjAzMywicGF0aCI6Ii8yMDc1NzMxOS8zNTM5MzI3OTgtOTI2YWViMDItOWIzZC00MmJkLWI2ODQtOWY2Y2RmNDkxZWJlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzMxVDE4NTM1M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTg4NjVkZjk2MTk3ZmM0MjZiYTY3YmRkMDJhMjA5OGExNmRmZWI0ODgzZmNjYWMyYjY1NzliOTU0MGJlNzBiNDAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.UcF6EJdOmurxPovi1QLeMbMwmxxAkubwF36A-qj-Dyg)

No, you can't close. It's still happening on my end, even after several macOS updates and IINA versions.

**Comment by axu2** [comment]
Actually, I was wrong. Weird stuff is happening with this. I can scroll horizontally while watching a video and nothing happens. But once my mouse hovers over the video scrollbar, it starts scrolling through the video like my horizontal scroll was buffered.

Edit: Looks my complaints are already detailed.
